Inc/stm32f446xx_i2c.h

```diff
@@ -136,22 +130,12 @@ void I2C_Init(I2C_Handle_t *pI2CHandle);									//INITIALIZES I2C PORT
 void I2C_DeInit(I2C_RegDef_t *pI2Cx);										//DEINITIALIZES I2C PORT
 
 /*
- *BLOCKING		 DATA TRANSMIT / RECIEVE
+ * DATA TRANSMIT / RECIEVE
  */
 
 void I2C_MasterSendData(I2C_Handle_t *pI2CHandle, uint8_t *pTXBuffer, uint32_t Len, uint8_t SlaveAddr);
 void I2C_MasterRecieveData(I2C_Handle_t *pI2CHandle, uint8_t *pRXBuffer, uint32_t Len, uint8_t SlaveAddr);
 
-
```

Comment:
Add interrupt data transmit ? receive also

Reply:
done ,idk what happned

Reply:
i had to change  the whole i2c after that I2C_Handle_t basically